torchaudio/models/conv_tasnet.py

```diff
@@ -159,7 +159,7 @@ def forward(self, input: torch.Tensor) -> torch.Tensor:
             output = output + skip
         output = self.output_prelu(output)
         output = self.output_conv(output)
-        output = torch.sigmoid(output)
+        output = self.mask_activate(output)
```

Comment:
This is a change in behavior, is it expected?

Reply:
According to Asteroid implementation, the activation function is relu for LibriMix data, and the performance is better than sigmoid. 
Maybe I can set a parameter to let users choose which activation function to use?